lib/src/compile.ts

```diff
@@ -72,16 +79,24 @@ function newCompileRequest(options: {
   return request;
 }
 
+const mapSyntaxes = {
+  indented: 'INDENTED',
+  scss: 'SCSS',
+  css: 'CSS',
+} as const;
+
```

Comment:
I'm not sure this map is adding much. Recommend in-lining this as a 3-way `if` statement in line 99